documentation/working-with-testcafe-studio/running-tests.md

```diff
@@ -60,24 +60,24 @@ Follow the steps below to run a test in remote browsers:
     ![Remote Browser Configuration](../../images/working-with-testcafe-studio/remote-browser-configuration.png)
 
 2. Select the configuration in the **Run Configuration** list.
-3. Start the test run by clicking the **Run test** button. TestCafe provides URLs to open in the remote browsers which you want to test. When you open these URLs, the browser connects to the TestCafe Studio server and starts testing.
+3. Click the **Run test** button to start the test. TestCafe provides URLs to open in the remote browsers which you want to test. When you open these URLs, the browser connects to the TestCafe Studio server and starts testing.
 
 ![Running in Remote Browsers](../../images/working-with-testcafe-studio/running-in-remote-browsers.png)
 
 ## Concurrent Test Execution
 
-To save time spent on testing, TestCafe allows you to execute tests concurrently. *Concurrency* is an optional mode that allows you to invoke multiple instances of the same browser. These instances constitute the pool of browsers against which tests run concurrently, i.e. each test runs in the first free instance.
+TestCafe allows you to execute tests concurrently. *Concurrency* is an optional mode that allows you to invoke multiple browser instances. Each test runs in the first available instance.
 
-To enable concurrency and specify a number of browser instances, use the **Concurrency** option of the run configuration's settings (see [Advanced Options](../user-interface/run-configurations-dialog.md#advanced-options)).
+To enable concurrency and specify the number of browser instances, use the run configuration's **Concurrency** option (see [Advanced Options](../user-interface/run-configurations-dialog.md#advanced-options)).
 
-> Concurrent test execution is not supported in Microsoft Edge. This is because there is no known way to start Edge in a new window and make it open a particular URL.
+> Microsoft Edge does not support concurrent tests. This is because there is no known way to start Edge in a new window and make it open a URL.
 
-You can also use concurrency when testing against multiple browsers.
+You can also use concurrency when testing multiple browsers.
```

Comment:
@dirk-pieterse 

We are testing not browsers, but a webpage in multiple browsers.

Reply:
Testing against? Testing what against? Do you mean 'You can check a webpage in multiple browsers simultaneously/concurrently'? No need to add 'use concurrency when'.

Reply:
We can rewrite this phrase as: "You can also use the Concurrency option when testing a webpage in multiple browsers." Is it good?

In TestCafe Studio, *simultaneously* is not *concurrently*. *Simultaneously* means that you can perform testing in multiple browsers at the same time. *Concurrently* means that there are several instances of each browser. For example, we specify 4 browsers and set the Concurrency option to 4, this means that there are 16 browser instances in the testing process.

Reply:
"You can also use the Concurrency option when testing a webpage in multiple browsers." is good